src/System.Data.Common/src/System/Data/Common/DbConnection.cs

```diff
@@ -35,6 +35,8 @@ protected DbConnection() : base()
         /// </summary>
         protected virtual DbProviderFactory DbProviderFactory => null;
 
+        internal DbProviderFactory ProviderFactory => DbProviderFactory; // This property is required for DbProviderFactories' API and will appear as 'dead' code till that class has been ported.
```

Comment:
You could get rid of the comment now that the consumer of this property is being brought in